tracer/tracer.go

```diff
@@ -279,11 +279,20 @@ func initializeMapsAndPrograms(kmod *kallsyms.Module, cfg *Config) (
 	// References to eBPF maps in the eBPF programs are just placeholders that need to be
 	// replaced by the actual loaded maps later on with RewriteMaps before loading the
 	// programs into the kernel.
-	coll, err := support.LoadCollectionSpec(cfg.DebugTracer)
+	coll, err := support.LoadCollectionSpec()
 	if err != nil {
 		return nil, nil, fmt.Errorf("failed to load specification for tracers: %v", err)
 	}
 
+	with_debug_output := uint32(0)
+	if cfg.DebugTracer {
+		with_debug_output = 1
+	}
+
+	if err = coll.Variables["with_debug_output"].Set(with_debug_output); err != nil {
+		return nil, nil, fmt.Errorf("failed to set debug output: %v", err)
+	}
```

Comment:
nit: would this look better?

```suggestion
	if cfg.DebugTracer {
		if err = coll.Variables["with_debug_output"].Set(uint32(1)); err != nil {
			return nil, nil, fmt.Errorf("failed to set debug output: %v", err)
		}
	}
```